app/assets/javascripts/home.coffee

```diff
@@ -38,7 +43,30 @@ set_coords = (position) ->
         Sentry.setExtra("response_status",  statusText)
         Sentry.setExtra("response_error",  errorText)
         Sentry.captureException(err)
-      
+
+set_coords_by_latlng = (latlng) ->
+  $('submission_latitude').attr 'value', latlng.lat
+  $('submission_latitude').attr 'value', latlng.lng
+  $.ajax
```

Comment:
A lot of this function is similar to set_cords. Could they be merged and the success handler vary for each context?